CONTRIBUTING.md

```diff
@@ -1,48 +1,53 @@
 # Contributing
- 
+
 Cool, that you are interested in contributing. We like pull request and suggestions from everyone.
 
 If you are planning in making a major contribution we appreciate your opening an issue or contacting [us](mailto:codecharta@github.com).
 
-Some things that will increase the chance that your pull request is accepted:
+Some things that will increase the chance of that your pull request to be accepted:
 
-Write clean code.
-Write tests.
-Write a good commit message.
-Code for Analysis sub-project only in Java and Kotlin languages.
+- Write clean code.
+- Write tests for your new code and regression tests after fixing a bug.
+- Write a good commit message and include the issue number at the end `#123`.
+- Code for Analysis sub-project only in Java and Kotlin languages.
```

Comment:
```suggestion
```

That seems rather obvious?

Reply:
Totally. That line actually confused me and I wasn't sure if I misunderstood something lol